src/backend/u64/field.rs

```diff
@@ -276,6 +275,14 @@ impl Mul<FieldElement> for FieldElement {
     }
 }
 
+impl<'a> From<&'a Ristretto255Scalar> for FieldElement {
```

Comment:
This impl should be removed as it is also declared on L:143 and the lib does not compile due to this error. 

See: 
```
error[E0119]: conflicting implementations of trait `std::convert::From<&curve25519_dalek::scalar::Scalar>` for type `backend::u64::field::FieldElement`:
   --> src/backend/u64/field.rs:278:1
    |
143 | impl<'a> From<&'a Ristretto255Scalar> for FieldElement {
    | ------------------------------------------------------ first implementation here
...
278 | impl<'a> From<&'a Ristretto255Scalar> for FieldElement {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `backend::u64::field::FieldElement`
```

Reply:
Solved from my branch.